frontend/src/RecordingPageComp/RecordingSelectionComp.jsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/prop-types */
+
+import { useContext, useEffect, useState } from "react";
+import { UserContext } from "../UserContext.js";
+
+function RecordingSelector({ setSelectedBlob, selectedBlob }) {
+  const { user } = useContext(UserContext);
+  const [recordings, setRecordings] = useState([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    const container = document.querySelector(".recording-container");
+    container.addEventListener("scroll", handleScroll);
+    return () => {
+      container.removeEventListener("scroll", handleScroll);
+    };
```

Comment:
Why do you need to add and remove the event listener? Also can you not add the event listener to the div element directly on line 70 (i.e. `<div onscroll="function()">`?